archunit/src/test/java/com/tngtech/archunit/core/domain/JavaClassTest.java

```diff
@@ -2163,12 +2157,6 @@ public boolean matches(JavaMember value) {
         };
     }
 
-    private static JavaClass fakeClassWithPackage(String pkg) {
-        JavaClass javaClass = mock(JavaClass.class);
-        when(javaClass.getPackageName()).thenReturn(pkg);
-        return javaClass;
-    }
-
```

Comment:
Did you intend to add this cleanup to the following commit?
> mark classes that are part of the public API as `@PublicAPI`

Reply:
Yes, because making `JavaClass` `final` was the reason to do this. But I guess I could also split it out (or motivate it in the commit message).

Reply:
I split it into a separate commit so it's easier to understand the motivation :slightly_smiling_face: